readme.md

```diff
@@ -726,6 +726,7 @@
 - [ZeroNet](https://github.com/zolagonano/awesome-zeronet#readme) - A decentralized web-like network of peer-to-peer users.
 - [Cosmos SDK](https://github.com/cosmos/awesome-cosmos#readme) - Modular framework for building app-specific blockchains in Go.
 - [Tor](https://github.com/polycarbohydrate/awesome-tor#readme) - A free overlay network for enabling anonymous communication.
+- [ATProto](https://github.com/atblueprints/awesome-atproto#readme) - Open, decentralized network for building social applications.
```

Comment:
Must not feature `Contributing` or `Footnotes` sections.

Reply:
@Kristories It's not in the table of contents, as that point refers to.

![image](https://github.com/user-attachments/assets/68423bd5-0743-48b8-beed-d6a97079ba5a)